conans/client/grapher.py

```diff
@@ -0,0 +1,39 @@
+import os
+from conans.model.ref import ConanFileReference
+
+class ConanGrapher(object):
+    def __init__(self, project_reference, deps_graph):
+        self._deps_graph = deps_graph
+        self._project_reference = project_reference
+
+    def graph(self):
```

Comment:
The positional arguments get messed up if I do this because of argparse interpreting the positional argument `reference` as the optional argument to `--graph`.

For example:

`conan info --graph=graph.dot ..` works because it interprets `graph.dot` as the graph output filename and `..` as the positional argument referring to the location of the conanfile in question.

But:

`conan info --graph ..` fails because argparse thinks that `..` is the argument to `--graph` rather than the positional argument to the conanfile.

This is a limitation with argparse I believe, when attempting to have an optional argument with an optional value.